maven/bnd-export-maven-plugin/src/main/java/aQute/bnd/maven/export/plugin/ExportMojo.java

```diff
@@ -64,6 +64,9 @@ private void export(File runFile) throws Exception {
 			return;
 		}
 		try (Bndrun run = Bndrun.createBndrun(null, runFile)) {
+			String bndrun = getNamePart(runFile);
+			File bndrunBase = new File(targetDir, "export/" + bndrun);
```

Comment:
I think the call to `exportRunbundles` needs to change to use `bndrunBase` instead of `targetDir` as the argument. You do not want all the runbundles of all the bndrun file to dump their jars into one folder.